public/source/main.js

```diff
@@ -41,12 +41,9 @@ var adBlocker = require('just-detect-adblock');
       clearTimeout(id);
     };
 
-  programEvents.add(function(event) {
-    if (event.event === 'init-tooltip') {
-      $('.bootstrap-tooltip').tooltip().removeClass('bootstrap-tooltip');
-    }
+  $(document).tooltip({
+    selector: '.bootstrap-tooltip'
```

Comment:
Instead of the class, can/should we use `[data-toggle="tooltip"]`?

I think the class was only used (and added) so there was something to distinguish the initialized from the uninitialized tooltips.

Of course, that change could be a separate PR

Reply:
Yes, following the bootstrap standard `[data-toggle="tooltip"]` selector would be good.